src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const isPerson = sex === 'm';
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && (
+        <p className="Person__age">{`I am ${age}`}</p>
+      )}
+
+      {isMarried ? (
+        <p className="Person__partner">
+          {`${partnerName} is my ${isPerson ? 'wife' : 'husband'}`}
+        </p>
+      ) : (
+        <p className="Person__partner">I am not married</p>
+      )}
```

Comment:
You can shorten your code and change only the content of the paragraph, rather than re-rendering several under different conditions
```suggestion
     {!isMarried
          ? 'I am not married'
          : `${partnerName} is my ${partner}`
        }
```